domains/japlic.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "japlic",
+    "discord": "1125587946087649310",
+  },
+  "record": {
+    "CNAME": "https://japlic.github.io/iamadev/"
```

`CNAME` record must contain a domain name only,
```suggestion
    "CNAME": "japlic.github.io"
```